Documentation/k8s-monitoring.md

```diff
@@ -64,3 +64,13 @@ kubectl -n rook delete statefulset prometheus-rook-prometheus
 kubectl delete -f https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.8/bundle.yaml
 ```
 Then the rest of the instructions in the [Prometheus Operator docs](https://github.com/coreos/prometheus-operator#removal) can be followed to finish cleaning up.
+
+## Special Cases
+
+### Tectonic Bare Metal
+In order to be able to monitor the Rook cluster on a Tectonic Bare Metal installation, you will need to create a ServiceMonitor.
+This ServiceMonitor can be tracked by a Prometheus instance which can be created using the [Prometheus Operator](https://coreos.com/operators/prometheus/docs/latest/). Tectonic strongly discourages the `tectonic-system` Prometheus instance to be used outside their intentions, so you need to create a new Prometheus instance yourself.
```

Comment:
Please add that there are example files for that in the https://github.com/rook/rook/tree/master/cluster/examples/kubernetes/monitoring directory.

Reply:
I just saw that this part is kind of a duplicate of this part above in the document: https://github.com/rook/rook/pull/1187/files/7891e0030f164fa6c1858b3845596ec569b6fe97#diff-83d97d501bf33a5aee3a6b28e675d756R27
Slimming it down to just note that Tectonic discourages `tectonic-system` Prometheus instance usage should be enough here.